ros/angel_system_nodes/angel_system_nodes/object_detector_with_descriptors.py

```diff
@@ -317,7 +301,7 @@ def postprocess_detections(self, rois, cls_prob, pooled_feat, im_scales):
                 objects = torch.argmax(scores[keep_boxes][1:])
                 objects = torch.unsqueeze(objects, 0)
                 box_dets = np.zeros((1, 4))
-                boxes = pred_boxes[keep_boxes.cpu()]
+                boxes = filled_boxes[keep_boxes.cpu()]
```

Comment:
Why is `keep_boxes` being transferred down to the CPU here? It's not elsewhere and `filled_boxes` seems to also be on the GPU given line 286 above.